packages/connectivity/src/scp-cf/destination/destination-from-service.ts

```diff
@@ -352,6 +322,16 @@ class DestinationFromServiceRetriever {
     return credentials[0];
   }
 
+  private async fetchDestinationWithoutToken(
+    jwt: string
+  ): Promise<Destination | undefined> {
+    return fetchDestinationByNameWithoutTokens(
```

Comment:
[req] If the outer function is called`...WithoutToken` why is the inner `...WithoutTokenSSSS`?